libraries/abstractions/ble_hal/test/include/iot_test_ble_hal_common.h

```diff
@@ -83,8 +83,8 @@ typedef struct
 
 #define bletestsDEFAULT_CHAR_VALUE       "hello"
 #define bletestsSTRINGYFIED_UUID_SIZE    36 /* like "8a7f1168-48af-4efb-83b5-e679f9320002" */
-#define bletests_LONG_WRITE_LEN          210
```

Comment:
Is the above #define referring to "stringy field"? What is that?

Reply:
it refers to the data length that we selected for long write test case

Reply:
It would probably be better written as `bletestsSTRINGIFIED_UUID_SIZE` or `bletestsUUID_STRING_LENGTH`.